app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,24 @@
+class GroupsController < ApplicationController
+skip_before_action :authenticate_user!
```

Comment:
no need to write tests but please remove this line

Reply:
Done